### ISSUE TITLE
Output problem

### ISSUE THREAD

**Issue by sebastianopazo1** [ShirleyMaxx/VirtualMarker#17]
Hi! Great work with the project. I'm having some issues with the output when running the project. 
I installed the environment as specified but the output video is exactly the same as the input. I have no warnings or errors about the model or missing files. 
thanks for the help!
Output video: 

https://github.com/ShirleyMaxx/VirtualMarker/assets/53013616/77ec569d-b2cd-4aa4-a6ef-f8f3c350b9c3

Output of the console when running the project:

sh command/simple3dmesh_infer/baseline.sh
args: Namespace(batch_size=32, cfg='./configs/simple3dmesh_infer/baseline.yml', cur_path='.', data_path='.', device=[0], experiment_name='simple3dmesh_infer/', gpus=1, input_path='inputs/input.mp4', input_type='video', is_distributed=False, seed=123)
Experiment Data on ./experiment/simple3dmesh_infer/exp_11-24_07_25
Input path: inputs/input.mp4
Input type: video
Running "ffmpeg -i inputs/input.mp4 -r 30 -f image2 -v error /home/seba/Documents/VirtualMarker/inputs/input/%06d.png"
Images saved to "/home/seba/Documents/VirtualMarker/inputs/input"
=> load backbone statedict from ./VirtualPose/output/mix_coco_muco/multi_person_posenet_152/coco_backbone_res152_coco_muco/backbone_res152_mix_muco/model_last.pth.tar
  0%|                                                    | 0/32 [00:00<?, ?it/s]/home/seba/anaconda3/envs/virtualmarkers/lib/python3.8/site-packages/torch/nn/_reduction.py:42: UserWarning: size_average and reduce args will be deprecated, please use reduction='sum' instead.
  warnings.warn(warning.format(ret))
/home/seba/anaconda3/envs/virtualmarkers/lib/python3.8/site-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /opt/conda/conda-bld/pytorch_1695392020195/work/aten/src/ATen/native/TensorShape.cpp:3526.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
100%|███████████████████████████████████████████| 32/32 [00:27<00:00,  1.18it/s]
=> init HRNet weights from normal distribution
=> loading pretrained HRNet model models/pytorch/pose_coco/pose_hrnet_w48_384x288.pth
=> reinit final layer...
=> successfully loaded
==> Loading checkpoint
Fetch model weight from experiment/final.pth.tar
Successfully load checkpoint from experiment/final.pth.tar.
===> Start inferencing...
100%|█████████████████████████████████████████████| 8/8 [00:03<00:00,  2.64it/s]
100%|█████████████████████████████████████████| 253/253 [00:04<00:00, 51.32it/s]
Results saved to ./experiment/simple3dmesh_infer/exp_11-24_07_25/vis.
===> Done.

**Comment by ShirleyMaxx** [comment]
Hi, could you please check whether the rendering environment is correctly installed? I guess it is still a problem due to the incorrect environment. You may refer to this [issue](https://github.com/ShirleyMaxx/VirtualMarker/issues/12).
